code-challenges.test.js

```diff
@@ -21,39 +21,143 @@ const numbersArray2 = [24, 27, 30, 33, 36]
+      expect(divisBy3(object3)).toEqual("-7 is not divisible by three")
+      expect(divisBy3("hello")).toEqual("error")
+      expect(divisBy3({number: "hello"})).toEqual("error")
+    })
+})
+
+//  ReferenceError: divisBy3 is not defined
 
 // b) Create the function that makes the test pass.
 
+// Pseudocode:
+//   create a function called divisBy3
+//      takes in an object as an argument
+//          iterates the objects number through a conditional statement that checks if the remainder of the object divided by three equals 0
+//              if the data type of the argument is not an object OR the datatype of the object.number is not a number ---- return 'error'
+//              else if the remainder of object.number divided by 3 is 0 ---- return `${object.number} is divisible by three`
+//              else return `${object.number} is not divisible by three`
+
+const divisBy3 = (object) => {
+    if (typeof object != "object" || typeof object.number != "number"){
+        return "error"
+    } else if (object.number % 3 === 0){
+        return `${object.number} is divisible by three`
+    } else {
+        return `${object.number} is not divisible by three`
+    }
+}
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+
+
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+describe("arrayCapitalizer", () => {
+    const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+    // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+    const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+    // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+    it("takes in an array of words and returns an array with all the words capitalized", () => {
+      expect(arrayCapitalizer(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+      expect(arrayCapitalizer(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+      expect(arrayCapitalizer(["hello", "i", "am", "the", "wizard"])).toEqual(["Hello", "I", "Am", "The", "Wizard"]) //this one is just for fun, inside joke in our breakout room, im not the wizard lol
+    })
+})
+
+// ReferenceError: arrayCapitalizer is not defined
 
 // b) Create the function that makes the test pass.
 
+// Pseudocode: (this pseudo is pretty bloated, but I want to share that I know whats going on in my code. If you would rather I not do this in the future please let me know)
+
```

Comment:
This is great! It explains your thought process and I feel like I could code this out just by reading the pseudocode. Nicely done!